templates/inspec/examples/google_compute_disk/google_compute_disk.erb

```diff
@@ -1,19 +1,20 @@
 <% gcp_project_id = "#{external_attribute('gcp_project_id', doc_generation)}" -%>
 <% gcp_zone = "#{external_attribute('gcp_zone', doc_generation)}" %>
-<% gcp_compute_disk_name = "#{external_attribute('gcp_compute_disk_name', doc_generation)}" -%>
-<% gcp_compute_disk_image = "#{external_attribute('gcp_compute_disk_image', doc_generation).gsub('\'', '')}" -%>
-<% gcp_compute_disk_type = "#{external_attribute('gcp_compute_disk_type', doc_generation)}" -%>
+<% snapshot = grab_attributes['snapshot'] -%>
```

Comment:
What's going on here?

Reply:
Redoing this test to not use an existing disk from other tests that recently had a random suffix added to it. This causes VCR to fail unless I hook in and set a seed for the random generator, which doesn't sound like a good idea